setup.py

```diff
@@ -52,11 +39,18 @@ def get_ucp_version():
     ),
 ]
 
-cmdclass = dict(build_ext=build_ext)
-cmdclass = versioneer.get_cmdclass(cmdclass)
+cmdclass = dict()
+cmdclass.update(versioneer.get_cmdclass())
+cmdclass["build_ext"] = build_ext
```

Comment:
Why is this needed? The former should already use `build_ext`. Please see [this doc]( https://github.com/python-versioneer/python-versioneer/blob/83f20fdf886df7a6089fca3bb7fc3c33198bd629/INSTALL.md?plain=1#L132 )

Reply:
Hmm I'm not sure why I did this. May have been a merge from an old commit. I'll undo this change.